.travis.yml

```diff
@@ -6,10 +6,6 @@ node_js:
 sudo: false
 dist: trusty
 
-branches:
```

Comment:
can we add this back in? It will keep our Travis from slowing down too much 👍

Reply:
sure